refine aurora doc (#6144)
pingcap/docs: data-migration/aurora/migrate-aurora-overview.md

# Overview of migration from Aurora to TiDB

> This document introduces how to migrate full data from Amazon Aurora MySQL to TiDB. Here is a list of tools that may be used and the differences

Reviewer: and the differences 好像不完整？建议优化一下。另外句子改成主动语态。Here is a list of tools that you can use

|Tool|Data Export|Data Import|Increment Sync|Export Speed| Import Speed|
|:-: |:-:|:-:|:-:|:-:|:-:|
|[Dumpling](https://github.com/pingcap/dumpling)|Yes|No|No|Fast|-|
|[Lightning](/tidb-lightning/tidb-lightning-overview.md)|No|Yes|No|-|Fast|
|[Data Migration](https://github.com/pingcap/dm) (DM)|Yes|Yes|Yes|Fast| Normal|

> Using existing snapshot in Aurora is the most convenient way. However, [DM](https://github.com/pingcap/dm) does not yet support the parquet format, so you need to do the following:

Reviewer 2: ```suggestion
Using the existing snapshot in Aurora is the most convenient way. However, [DM](https://github.com/pingcap/dm) does not yet support the parquet format, so for different task mode, you need to use different tools as follows:
```

> Using existing snapshot in Aurora is the most convenient way. However, [DM](https://github.com/pingcap/dm) does not yet support the parquet format, so you need to do the following:

Author: 客户一般会定时创建快照用于备份目的；所以我们可以利用已经创建的快照，而不用强制用户创建新快照

> Using Aurora's existing snapshot is the most convenient way, but beacuse [DM](https://github.com/pingcap/dm) does not yet support the parquet format, so we need to 

Reviewer: ```suggestion
Using existing snapshot in Aurora is the most convenient way. However, [DM](https://github.com/pingcap/dm) does not yet support the parquet format, so you need to do the following:
```

1. Use Dumpling for schema export.

> 2. Use Lightning for full schema and data import. 

Reviewer: ```suggestion
2. Use TiDB Lightning for full schema and data import. 
```

> 3. Use DM for incremental data synchronization.

Reviewer: ```suggestion
3. Use DM for incremental data replication.
```
